client/src/controllers/OrderableController.ts

```diff
@@ -30,7 +31,7 @@ export class OrderableController extends Controller<HTMLElement> {
   static values = {
     animation: { default: 200, type: Number },
     container: { default: '', type: String },
-    message: { default: '', type: String },
+    messages: { default: '{}', type: String },
```

Comment:
```suggestion
    messages: { default: {}, type: Object },
```

If you set this to `Object`, Stimulus will automatically parse as json when you access the value. 

See https://stimulus.hotwired.dev/reference/values#types